gen/config/downloads.yaml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
> It won't be difficult at all to extract the version specifiers for the different repositories. I had a vague plan to extract version numbers to a config of their own in any case, in order that you wouldn't need to always specify the correct version at the command line invocation and the repository would be self-documenting on which version of files were being used.

Yeah, having a list of source data versions in a config files sounds the way to go!